configs/defs/rocky.textproto

```diff
@@ -0,0 +1,1179 @@
+      "[Service]\n"
+      "ExecStart=-/usr/lib/systemd/systemd-sulogin-shell rescue"
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      " check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/usr/lib/systemd/system/rescue.service\"}}"
+      "      files_to_check:{files_in_dir:{dir_path:\"/etc/systemd/system/rescue.service.d\",recursive:true}}"
+      "      content_entry:{"
+      "        match_type: ALL_MATCH_ANY_ORDER"
+      "        match_criteria: {"
+      "          filter_regex: \".*/systemd-sulogin-shell.*\""
+      "          expected_regex: \"ExecStart=/usr/lib/systemd/systemd-sulogin-shell rescue\""
+      "        }"
+      "      }"
+      "    }"
+      " }"
+      "}"
+  }
+}
+benchmark_configs: {
+  id: "systemwide-crypto-policy-is-not-legacy"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/o:rockylinux:rockylinux:8.5" version: "1.0.0" benchmark_document: "CIS Rocky Linux 8" }
+    title: "Ensure system-wide crypto policy is not legacy"
+    description:
+      "The system-wide crypto-policies followed by the crypto core components allow "
+      "consistently deprecating and disabling algorithms system-wide.\n"
+      "The individual policy levels (DEFAULT, LEGACY, FUTURE, and FIPS) are included in the "
+      "crypto-policies(7) package."
+    rationale:
+      "If the Legacy system-wide crypto policy is selected, itincludes support for TLS 1.0, TLS 1.1, "
+      "and SSH2 protocols or later. The algorithms DSA, 3DES, and RC4 are allowed, while RSA "
+      "and Diffie-Hellman parameters are accepted if larger than 1023-bits.\n"
+      "These legacy protocols and algorithms can make the system vulnerable to attacks, "
+      "including those listed in RFC 7457\n"
+      "Impact:\n"
+      "Environments that require compatibility with older insecure protocols may require the use "
+      "of the less secure LEGACY policy level."
+    remediation:
+      "Run the following command to change the system-wide crypto policy\n"
+      "# update-crypto-policies --set <CRYPTO POLICY>\n"
+      "Example:\n"
+      "# update-crypto-policies --set DEFAULT\n"
+      "Run the following to make the updated system-wide crypto policy active\n"
+      "# update-crypto-policies\n"
+      "Default Value:\n"
+      "DEFAULT\n"
+      "References:\n"
+      "1. CRYPTO-POLICIES(7)\n"
+      "2. https://access.redhat.com/articles/3642912#what-polices-are-provided-1"
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/etc/crypto-policies/config\"}}"
+      "      content_entry:{"
+      "        match_type: NONE_MATCH"
+      "        match_criteria: {"
+      "          filter_regex: \"\\\\S*LEGACY\\\\s* (\\\\s+#.*)?$\""
```

Comment:
Hmm, does this work without an expected_regex being set? If it does then I guess we also don't need the expected_regex in other places where it's identical, e.g. the `expected_regex: \".* /home .* - .*\""` in home-nosuid

Reply:
Also, filter regexes implicitly include ^ and $, no need to add them separately.

Reply:
Oh, and the regex in the doc starts with `^\s*LEGACY\s*`, is there a reason we have \S here instead (uppercase)?